app/code/core/Mage/Directory/Model/Resource/Country/Collection.php

```diff
@@ -33,11 +33,6 @@
  */
 class Mage_Directory_Model_Resource_Country_Collection extends Mage_Core_Model_Resource_Db_Collection_Abstract
 {
-    /**
-     * @var Mage_Directory_Model_Country[]
-     */
-    protected $_items;
```

Comment:
podle mě to tu bylo schválně kvůli phpdoc
dokážeš tam dostat phpdoc bez přepsání tý proměnný?